environment.yml

```diff
@@ -10,10 +10,13 @@ dependencies:
   - matplotlib<3
   - python-stratify
   # Multi language support:
+  - gcc
   - ncl
   - ncurses=6.1=hfc679d8_1
   - r-base
   - r-rcpp
+  - r-ncdf4
```

Comment:
Please move dependencies to esmvaltool/install/R/r_requirements.txt if possible, I don't think it's a good idea to mix two installation methods.